tests/templates/message_templates/event_online_receipt_text.tpl

```diff
@@ -29,14 +29,9 @@ pay_later_receipt:::{$pay_later_receipt}
 event.event_title:::{$event.event_title}
 event.event_start_date:::{$event.event_start_date|crmDate:"%A"}
 event.event_end_date:::{event.end_date|crmDate:"%Y%m%d"}
-{if !empty($event.is_monetary)}
-event.is_monetary:::{$event.is_monetary}
-{/if}
+event.is_monetary:::{event.is_monetary|boolean}
 event.fee_label:::{event.fee_label}
-{if !empty($event.participant_role)}
-  event.participant_role::{$event.participant_role}
-  defaultRole:::{$defaultRole}
-{/if}
+event.participant_role::{event.participant_role_id:label}
```

Comment:
Why does everyone else get 3 colons but this only gets 2?

Reply:
@colemanw favouritism but OK I shared the colon love